spec/models/ext_management_system_spec.rb

```diff
@@ -51,40 +51,43 @@
-      "kubevirt"                    => "KubeVirt",
-      "openshift"                   => "OpenShift",
-      "openshift_monitor"           => "Openshift Monitor",
-      "openstack"                   => "OpenStack",
-      "openstack_infra"             => "OpenStack Platform Director",
-      "openstack_network"           => "OpenStack Network",
-      "lenovo_ph_infra"             => "Lenovo XClarity",
-      "nsxt"                        => "VMware NSX-T Network Manager",
-      "nuage_network"               => "Nuage Network Manager",
-      "redfish_ph_infra"            => "Redfish",
-      "redhat_network"              => "Redhat Network",
-      "rhevm"                       => "Red Hat Virtualization",
-      "scvmm"                       => "Microsoft System Center VMM",
-      "vmwarews"                    => "VMware vCenter",
-      "vmware_cloud"                => "VMware vCloud",
-      "vmware_cloud_network"        => "VMware Cloud Network",
-      "cinder"                      => "Cinder ",
-      "swift"                       => "Swift ",
+      "ansible_tower_automation"        => "Ansible Tower Automation",
+      "azure"                           => "Azure",
+      "azure_network"                   => "Azure Network",
+      "azure_stack"                     => "Azure Stack",
+      "azure_stack_network"             => "Azure Stack Network",
+      "ec2"                             => "Amazon EC2",
+      "ec2_network"                     => "Amazon EC2 Network",
+      "ec2_ebs_storage"                 => "Amazon EBS",
+      "embedded_ansible_automation"     => "Embedded Ansible Automation",
+      "s3"                              => "Amazon S3",
+      "foreman_configuration"           => "Foreman Configuration",
+      "foreman_provisioning"            => "Foreman Provisioning",
+      "gce"                             => "Google Compute Engine",
+      "gce_network"                     => "Google Network",
+      "ibm_cloud_networks"              => "IBM Cloud Networks",
+      "ibm_cloud_power_virtual_servers" => "IBM Cloud",
+      "ibm_cloud_storage"               => "IBM Cloud Storage",
+      "kubernetes"                      => "Kubernetes",
+      "kubernetes_monitor"              => "Kubernetes Monitor",
+      "kubevirt"                        => "KubeVirt",
+      "openshift"                       => "OpenShift",
+      "openshift_monitor"               => "Openshift Monitor",
+      "openstack"                       => "OpenStack",
+      "openstack_infra"                 => "OpenStack Platform Director",
+      "openstack_network"               => "OpenStack Network",
+      "lenovo_ph_infra"                 => "Lenovo XClarity",
+      "nsxt"                            => "VMware NSX-T Network Manager",
+      "nuage_network"                   => "Nuage Network Manager",
+      "redfish_ph_infra"                => "Redfish",
+      "redhat_network"                  => "Redhat Network",
```

Comment:
unrelated to this PR, but what is `Redhat Network`?

Reply:
It is Ovirt using OpenStack Neutron networking service, https://www.ovirt.org/develop/release-management/features/network/osn-integration.html

Reply:
so, should we rename it to something like `OSN (OpenStack Neutron)`?

Reply:
Since it is the child NetworkManager of a "Red Hat Virtualization" InfraManager it would be more consistent to call it "Red Hat Virtualization Network".  E.g. we don't call Openstack NetworkManager "Neutron" either [ref](https://github.com/ManageIQ/manageiq/pull/20483/files#diff-1f5be100b3b69417b854ff9a45a24fa3R78)

If we want the manager description to match the name of the underlying service we could certainly do that, most of these would be different then.